WordPressAuthenticator/Signin/LoginProloguePromoViewController.swift

```diff
@@ -71,14 +92,30 @@ class LoginProloguePromoViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setupBackground()
-        setupHeadingLabel()
+
+        if WordPressAuthenticator.shared.configuration.enableUnifiedCarousel {
```

Comment:
Nit: maybe we can do the change inside `setupHeadingLabel` (and renaming it to `styleHeadingLabel` too if it's more appropriate), so we don't change `viewDidLoad`?